compliant-standards.md

```diff
@@ -0,0 +1,150 @@
+# EMC Compliant Standards
+
+## 1. Introduction
+
+### 1.1 What is this document?
+
+The Email Markup Consortium (EMC) is working to improve email markup, the code used to structure and style content within an email. This document lays out the standards of what that markup should look like.  
+  
+This document is aimed at top level ideas of what the email markup should do, not specific ways to code.
+
+### 1.2 Who is this document for?
+
+This document is for anyone creating email markup, processing email markup, or displaying email markup. This includes (but isn’t limited to) email developers, email creation tools, email sending tools, and email clients.
+
+For this to work it will require developers to make improvements to the markup at the point of creation. As long as that markup is valid it should remain in a recognisable state with minimal changes through to the point the recipient opens the email. This means any processing of that markup by an ESP, mail server, email client or any other technology that touches it, should also follow the guidelines to preserve the standard.
```

Comment:
Many terms are used here. Do we need to link to the [glossary of terms](https://github.com/email-markup-consortium/email-markup-consortium/blob/main/glossary.md)?

Reply:
Yeah it's a good point I've added a few but it's hard to link directly to the terms. This should be easier when these pages are moved to the website.